bool got_cmf_should_plan
string got_cmd_refPos
string actPos
bool executing
bool planning
